benchmarks/benchmark_transform.py

```diff
@@ -0,0 +1,19 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# http://asv.readthedocs.io/en/latest/writing_benchmarks.html
+import numpy as np
+from scipy import ndimage as ndi
+from skimage import transform
+
+
+class TransformSuite:
+    """Benchmark for transform routines in scikit-image."""
+    def setup(self):
+        self.image = np.zeros((2000, 2000))
+        idx = np.arange(500, 1500)
+        self.image[idx[::-1], idx] = 255
+        self.image[idx, idx] = 255
+
+    def time_hough_line(self):
+        # Running it 10 times to achieve significant performance time.
```

Comment:
Does ASV not do this for you? I guess it times its own overhead too?

Reply:
Actually, for this function to run it took around 50ms, which is quite less as we are concerned with seeing its performance in future compared to what it is now. So when @jni met me at Scipy, he told me to run a loop.

Reply:
In principle, this should be equivalent to setting `TransformSuite.number = 10`.

Reply:
Well not quite. AFAICT, by default ASV will by default run the benchmark some number of times until roughly 100ms passes.

Reply:
Great! @abhishekarya286 then it seems I was wrong to suggest my approach. Let's just have simple (single run) benchmarks from now on! =)

Reply:
Sure @jni @TomAugspurger.Thanks for confirming.